mage_ai/data_preparation/templates/transformers/transformer_actions/column/add.py

```diff
@@ -0,0 +1,24 @@
+from mage_ai.data_cleaner.transformer_actions.base import BaseAction
+from mage_ai.data_cleaner.transformer_actions.constants import ActionType, Axis
+from mage_ai.data_cleaner.transformer_actions.utils import build_transformer_action
+from pandas import DataFrame
+
+if 'transformer' not in globals():
+    from mage_ai.data_preparation.decorators import transformer
+
+
+@transformer
+def execute_transformer_action(df: DataFrame, *args, **kwargs) -> DataFrame:
+    """
+    Execute Transformer Action: ActionType.ADD
+    """
+    action = build_transformer_action(
+        df,
+        type=ActionType.ADD,
+        arguments=[],  # Specify columns to add. Must have at most two columns
```

Comment:
This transformer action is to add column with udf. It doesn't necessarily need two columns here. The arguments and options depend on the udf. We currently don't show this action type in UI because we need to support select udf to get the right template. You can remove this template for now.

Reply:
Removed template